backend/src/main/java/sysc4806group25/monkeypoll/MonkeyPollApplication.java

```diff
@@ -12,4 +17,35 @@ public static void main(String[] args) {
         SpringApplication.run(MonkeyPollApplication.class, args);
     }
 
+    @Bean
```

Comment:
I assume we are leaving this for testing purposes, will be removed soon

Reply:
Yes exactly, just to help with testing for whoever will work with these changes next, and will be removed later.